benchmark/C/Savina/src/micro/Counting.lf

```diff
@@ -38,7 +38,16 @@
  ]]] */
  // [[[end]]]
 
-target C;
+target C {
+    /* [[[cog
+      if (threaded_runtime=="True"):
+          cog.outl(f"threads: {threads},")
```

Comment:
Should we take this PR as an opportunity to address https://github.com/icyphy/lingua-franca/issues/290?

Reply:
I think #290 is a bigger change and I would rather address this separately

Reply:
OK, fair enough!